src/content/docs/concepts/drivers.md

```diff
@@ -2,12 +2,12 @@
 title: Drivers
 ---
 
-A _driver_ is a Membrane program whose sole responsibility is to expose the functionality of third-party APIs in your Membrane account so that other programs can easily make use of it. You can see all the drivers maintained by the Membrane team in the IDE by clicking "FIND PROGRAMS" in the top right and searching "membrane".
+A _driver_ is a Membrane program that wraps a third-party API so that other programs can easily make use of it. You can see all the drivers maintained by the Membrane team in the IDE by clicking `🔎 SEARCH` in the top left and searching "membrane".
 
 There's nothing fundamentally special about a driver. It's a program just like any other. That means you can create your own drivers for you or other Membrane users to use.
```

Comment:
```suggestion
A _driver_ is a Membrane program that wraps a third-party API so that other programs can easily make use of it. 

There's nothing fundamentally special about a driver. It's a program just like any other. That means you can create your own drivers for you or other Membrane users to use.
```